### ISSUE TITLE
Question about data split in the caso of training in FMB dataset

### ISSUE THREAD

**Issue by iacopo97** [LiJiahang617/Road-Former#8]
Good morning, sorry to bother you. I would like to know since there is no validation set in FMB, which kind of data split are you using a validation split?

**Comment by jianxinhuang3** [comment]
The FMB dataset is divided into a training set and a testing set. You can refer to the paper _Multi-interactive Feature Learning and a Full-time Multi-modality Benchmark for Image Fusion and Segmentation_ and the corresponding code repository for more details.

**Comment by jianxinhuang3** [comment]
Since there is no validation set, we directly perform validation on the test set.
